.travis.yml

```diff
@@ -1,12 +1,15 @@
 sudo: false
 language: node_js
-node_js: stable
+node_js:
+    - '6'
 
 env:
   global:
     - CXX=g++-4.9
     - secure: "VDsxy30sE9ivdqoXkaKXo0czbS4brNpwKEIblu7f1gVLx7OD9pjTc78cdwrVbZDBYroSiYVYuUrLDjpVjH88lL/LxRrru3V0CIlAqqa+ssXcqycCaT/6ds+ZymuTTGRh+Mf12pIKO+yc8jTov2M7AzPJdpS+ORP5dImYyE3ex9s="
 
+script: ./test_type_defs.sh
```

Comment:
It's the first time I use `node` on travis and I felt that something was wrong, too. Thankfully @leebyron pointed out that `npm install` and then `npm test` is automatically ran by travis. There's no need to specify a script.